compiler/rustc_codegen_cranelift/Cargo.lock

```diff
@@ -470,3 +506,23 @@ name = "windows_x86_64_msvc"
 version = "0.48.5"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "ed94fce61571a4006852b7389a063ab983c02eb1bb37b47f8272ce92d06d9538"
+
+[[package]]
+name = "zerocopy"
+version = "0.7.32"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "74d4d3961e53fa4c9a25a8637fc2bfaf2595b3d3ae34875568a5cf64787716be"
+dependencies = [
+ "zerocopy-derive",
+]
+
+[[package]]
+name = "zerocopy-derive"
```

Comment:
This shouldn't be listed in Cargo.lock: The derive feature of zerocopy shouldn't be enabled, and CI confirms that zerocopy-derive is indeed not built.

Reply:
I think this is a symptom of https://github.com/rust-lang/cargo/issues/10801. The crate appears in Cargo.lock but is not actually built in practice.